lib/plugins/slstats/slstats.js

```diff
@@ -38,17 +38,22 @@ class SlStats {
     const statsDisabledFilePath = path.join(serverlessDirPath, 'stats-disabled');
     const statsEnabledFilePath = path.join(serverlessDirPath, 'stats-enabled');
 
-    if (this.options.enable && !this.options.disable) {
-      if (this.serverless.utils.fileExistsSync(statsDisabledFilePath)) {
-        fse.renameSync(statsDisabledFilePath, statsEnabledFilePath);
+    try {
+      if (this.options.enable && !this.options.disable) {
+        if (fse.lstatSync(statsDisabledFilePath).isFile()) {
+          fse.renameSync(statsDisabledFilePath, statsEnabledFilePath);
+        }
+        this.serverless.cli.log('Stats successfully enabled');
       }
-      this.serverless.cli.log('Stats successfully enabled');
-    }
-    if (this.options.disable && !this.options.enable) {
-      if (this.serverless.utils.fileExistsSync(statsEnabledFilePath)) {
-        fse.renameSync(statsEnabledFilePath, statsDisabledFilePath);
+      if (this.options.disable && !this.options.enable) {
+        if (fse.lstatSync(statsEnabledFilePath).isFile()) {
+          fse.renameSync(statsEnabledFilePath, statsDisabledFilePath);
+        }
+        this.serverless.cli.log('Stats successfully disabled');
       }
-      this.serverless.cli.log('Stats successfully disabled');
+    } catch (e) {
+      throw new this.serverless.classes
+      .Error(`slstats failed. The following message: ${e.message}`);
```

Comment:
Could you update this to `Enabling / Disabling of statistics failed: ${error.message}`.